static/src/javascripts/projects/common/modules/discussion/user-avatars.js

```diff
@@ -13,14 +13,21 @@ define([
             updating = bonzo(bonzo.create('<div class="is-updating"></div>')),
             avatar = bonzo(bonzo.create('<img class="user-avatar__image" alt="" />')),
             userId = container.data('userid');
+
         container
```

Comment:
@sndrs thanks. How important is this here? The code is written to rely on the return value of the bonzo create operations, whereas fastdom seems to return simply an op identifier. The profile page this impacts has hardly any dom elements on it (https://profile.theguardian.com/public/edit) and I'm a bit reluctant to start rewriting existing code here unless absolutely necessary.